templates/services/app.tmpl.yaml

```diff
@@ -63,7 +63,7 @@ spec:
             name: app-config
         - secretRef:
             name: app-secret
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
I hope there is a really good reason for this, because it makes deployment slower.

Reply:
We have switched to `latest` versions of the images, so we do need this. But I believe an image will only be fetched if it changed